website/public/js/controllers/inventoryCtrl.js

```diff
@@ -211,5 +211,25 @@ habitrpg.controller("InventoryCtrl",
           break;
       }
     };
+
+    $scope.$on("habit:keyup", function (e, keyEvent) {
+      if (keyEvent.keyCode == "27") {
+        $scope.deselectItem();
+      }
+    });
+ //$scope.listen(keyEvent)
```

Comment:
Do we care if selectedWhatever has a value before setting it to null? Do we need these if statements?